lib/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+plugins {
+    // Apply the org.jetbrains.kotlin.jvm Plugin to add support for Kotlin.
+    id("org.jetbrains.kotlin.jvm") version "1.4.20"
+
+    `java-library`
+    `maven-publish`
+}
+
+repositories {
+    // Use JCenter for resolving dependencies.
+    jcenter()
+}
+
+dependencies {
+    // Align versions of all Kotlin components
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+
+    // Use the Kotlin JDK 8 standard library.
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+
+    // This dependency is used internally, and not exposed to consumers on their own compile classpath.
+    implementation("com.google.guava:guava:29.0-jre")
```

Comment:
we have this?

Reply:
I'll remove these two dependencies, they were generated among the project dependencies, i preferred to keep them for an initial clean setup.

Reply:
If you keep not needed things, it's not clean ;)